lib/french_toast/notifier.rb

```diff
@@ -4,16 +4,32 @@ def initialize(session_key)
       @session_key = session_key
     end
 
-    def notify(payload)
-      notify_by_action_cable(payload)
+    def notify(partial_name, payload = {})
+      processed_payload = process(partial_name, payload)
```

Comment:
Why do we pass the second argument? It doesn't appear to be used by the `#process` method.